tests/ui_tools/test_messages.py

```diff
@@ -1593,6 +1593,96 @@ def test_transform_content(self, mocker, raw_html, expected_content):
         rendered_text = Text(content)
         assert rendered_text.text == expected_content
 
+    @pytest.mark.parametrize(
+        "raw_html, expected_message_links",
+        [
+            (
+                """
+                <p><a href="https://github.com/zulip/zulip-terminal/pull/1">https://github.com/zulip/zulip-terminal/pull/1</a></p>
+                """,
+                {
+                    "https://github.com/zulip/zulip-terminal/pull/1": (
+                        "github.com",
+                        1,
+                        True,
+                    )
+                },
+            ),
+            (
+                """
+                <p><a href="https://foo.com">https://foo.com</a></p>
+                """,
+                {"https://foo.com": ("https://foo.com", 1, False)},
+            ),
+            (
+                """
+                <p><a href="#narrow/stream/206-zulip-terminal/topic/announce">https://chat.zulip.zulip/#narrow/stream/206-zulip-terminal/topic/announce</a></p>
```

Comment:
Why not chat.zulip.org? xD

Reply:
`SERVER_URL = "https://chat.zulip.zulip"` is defined on line 28 of `test_messages.py`, and I used it when calling `transform_content` in the test. Could be changed though, if needed :)